src/foam/core/FObjectArray.js

```diff
@@ -64,5 +64,39 @@ foam.CLASS({
         return foam.json.parse(value, prop.of, ctx);
       }
     }
+  ],
+
+  methods: [
+    function installInProto(proto) {
+      this.SUPER(proto);
+      var self = this;
+      Object.defineProperty(proto, self.name + '$push', {
+        get: function classGetter() {
```

Comment:
Why is this named classGetter?

Reply:
I did this to be consistent with types.js. I don't believe it changes the behaviour of anything so I assumed it was done for clarity. If you want this removed I can make another PR after to remove it from all of them.

Reply:
The name is given so that stack traces have meaningful names, but it was called classGetter when it was the getter, the name doensn't matter but it should be something meaningful or left out.